src/i18n/translations/messages_hi.json

```diff
@@ -61,7 +61,7 @@
   "starred": "तारांकित",
   "Enable notifications": "सूचना सक्षम करें",
   "Jot down something": "कुछ लिखिए",
-  "Message {recipient}": "संदेश {recipient}",
+  "Message {recipient}": "संदेश {प्राप्त करने वाला}",
```

Comment:
This shouldn't be translated as its for replacing, and here Message is signifying as a verb to send that person a message, so its better to be written as भेजें